### PR TITLE
Support parallel

### PR DESCRIPTION
**Pull request by ntvlogi**
Support parallel

### CODE REVIEW

File: src/main/java/com/automatedtest/sample/driver/TearDown.java [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
+        this.driver = Setup.currentDriver.get();
     }
 
     @After
     public void quitDriver(Scenario scenario) {
         if (scenario.isFailed()) {
             saveScreenshotsForScenario(scenario);
         }
-        this.driver.quit();
+        Setup.currentDriver.get().quit();
```
Comment (pklltt) [inline src/main/java/com/automatedtest/sample/driver/TearDown.java:22]: I don't think you have change this. it's already set to the correct driver in the constructor

**Review by pklltt — CHANGES_REQUESTED** [review]

File: src/main/java/com/automatedtest/sample/driver/TearDown.java (hunk 1 above)
Comment (ntvlogi) [reply src/main/java/com/automatedtest/sample/driver/TearDown.java]: yes, I reverted code.

File: pom.xml [hunk 2]
```diff
@@ -16,6 +16,16 @@
...
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.0</version>
+                <configuration>
+                    <parallel>classesAndMethods</parallel>
+                    <perCoreThreadCount>true</perCoreThreadCount>
+                    <threadCount>2</threadCount>
```
Comment (pklltt) [inline pom.xml:26]: set this as property, default to 1, so we can pass it as an argument

Comment (ntvlogi) [reply pom.xml]: change threadCount to 1

**Review by pklltt — APPROVED** [review]

**Pull request merged**